src/SapiEmitter.php

```diff
@@ -26,42 +26,48 @@ final class SapiEmitter
 
     private int $bufferSize;
 
-    public function __construct(?int $bufferSize = null)
+    public function __construct(int $bufferSize = null)
     {
         if ($bufferSize !== null && $bufferSize <= 0) {
             throw new \InvalidArgumentException('Buffer size must be greater than zero');
         }
         $this->bufferSize = $bufferSize ?? self::DEFAULT_BUFFER_SIZE;
     }
 
+    /**
+     * @param ResponseInterface $response
+     * @param bool $withoutBody
+     *
+     * @throws HeadersHaveBeenSentException
+     *
+     * @return bool
+     */
```

Comment:
The comment does not carry a semantic load, except for the @throws annotation
```suggestion
    /**
     * @throws HeadersHaveBeenSentException
     */
```